source/iface/math-array-types.rst

```diff
@@ -7,3 +7,564 @@
 ****************
 Math array types
 ****************
+
```

Comment:
Other sections have a link/reference to the SYCL spec.  Can that be added here?

Reply:
Added with https://github.com/KhronosGroup/SYCL_Reference/pull/145/commits/38d5460d79e12d1e98008c14f7f6a2bc6550a6d5